app/assets/stylesheets/forms.scss

```diff
@@ -67,3 +67,14 @@
 .vertical-drag {
   cursor: ns-resize;
 }
+
+.inset-text-field {
+  border-left: 5px solid #bcbcbc;
+  padding: 6px 0 6px 10px;
+  position: relative;
+  left: 25px;
+
+  label {
```

Comment:
We generally avoid element based selectors for styling, like `label`, preferring a class e.g. `.inset-text-field__label` but I'm not sure of the conventions here.

Reply:
Thanks Andy - added class